src/platform/intel/cavs/alternate_reset_vector.S

```diff
@@ -0,0 +1,311 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ */
+
```

Comment:
Probably best to add some more comments describing what this file does.

Reply:
Its cAVS based ASM code, I cannot add more comments than there was in cavs :( But most of the labels are self explanatory like "alt_boot_slave_core_idc_receiver"